packages/shared/scope-hoisting/src/link.js

```diff
@@ -438,6 +459,86 @@ export function link({
         );
         let mod = nullthrows(bundleGraph.getDependencyResolution(dep, bundle));
         path.replaceWith(t.valueToNode(mod.id));
+      } else if (callee.name === '$parcel$exportAll') {
```

Comment:
Not sure I totally understand why this is necessary vs the old `$parcel$exportWildcard` with a `$parcelRequire` inside. Seems like a bunch of logic got duplicated?